test/liquid_voting/delegations_test.exs

```diff
@@ -122,6 +142,13 @@ defmodule LiquidVoting.DelegationsTest do
       assert {:error, %Ecto.Changeset{}} = Delegations.create_delegation(args)
     end
 
+    # FAILING!!
+    # PR #109 https://github.com/liquidvotingio/api/issues/109
+    # "Allow creation of more than one proposal-specific delegation (different proposals) with same delegator and delegate"
+    # causes this failure.
+    # Addressed in issue #125:https://github.com/liquidvotingio/api/issues/125
+    # This test should pass when #125 is addressed as the global delegation should cause the deletion of the proposal-specific delegation.
```

Comment:
I think this test should actually expect something different after #125, because there won't be an error and the global delegation will be created. 

And we should assert that the original proposal-specific one will be deleted.

Maybe we can continue this discussion during implementation of #125, and just leave enough comments explaining that right now these aren't passing because of pending functionality to be implemented during #125, and that's it.

Reply:
Yes, my comments only explain why it fails now, and why it will pass again after #125. I will add to comments to explain we need new assertions after #125.

Reply:
No need to add to comments about future tests, it's really out of scope here. Let's just add the tests on #125.